### ISSUE TITLE
[Request] Publish the links to the Firefox Taskcluster so they are easily visible and accessible for users who want to test early software builds

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#11030]
Please share the builds with others who are willing to test early software builds. The links to the Firefox taskcluster are hidden and not mentioned and can't be found anywhere easily.

I stopped testing builds around when taskcluster links were changed a few months ago and only got them a while ago when someone shared this link with me: https://firefox-ci-tc.services.mozilla.com/tasks/index/project.mobile.fenix.v2 on Reddit. I have reported quite a lot bugs since having access to the builds early. Not only this you also don't publish compiled builds of Firefox Preview early on GitHub and only have the source code and only publish compiled builds once the Play Store also has it.

I request Mozilla to link the builds on GitHub `readme` page or somewhere else easily visible. If someone willingly wants to test the early builds then I think Mozilla shouldn't stop them. It is apparent that Mozilla should do this as experienced recently with the Play Store situation not having up-to-date builds (Play Store situation is okay recently, since I'm getting builds regularly now). Now I know I can have access to Nightly versions early and regularly when things get normal; that is not what I want; I want to have access anything that Firefox developers publish publicly and by the minute they publish it. Just like the following link has everything in it: https://firefox-ci-tc.services.mozilla.com/tasks/index/project.mobile.fenix.v2

Nowadays I only update my builds through the Taskcluster because it has every build; everything's up-to-date by the minute there.

For example I have the build Version `5.1.0-beta.2` which is neither available on the Play Store nor on GitHub (compiled builds). I'm willing to accept the risks of early software builds.

So please publish the links to the taskcluster builds. So that I don't lose access to the builds when you change the links to the taskcluster again in future.

Edit: I want to make it clear that I'm not saying that the builds themselves are hidden, what I'm saying is that the links to the Taskcluster aren't found easily anywhere. 

Edit:

tl;dr: in short what I'm saying is that please publish links to the Taskcluster builds somewhere that can be easily found. Like you can publish the links to them in GitHub's `README` page or somewhere easily visible and accessible. So users who want to test and install early software and help report bugs and get involved in the early in the development can do so.

**Comment by Dunexus** [comment]
I don't know if taskcluster builds are supposed to be shared to users. Nighly and beta channel on the playstore are available for this purpose i guess.
There has been a week or two of delayed nightly releases on the playstore because of international special situation but now it is back to normal and playstore nightlies are up to date.

**Comment by SS1113** [comment]
The builds can be accessed easily once I have the links to them. So I don't think they are not meant to be not accessed by public. 

Nightlies are okay now, but what if I want to test the Firefox Preview's beta versions before they are release on the Play Store, like the build for `5.1.0-beta.2` for example

**Comment by kbrosnan** [comment]
This is documented at https://github.com/mozilla-mobile/fenix/wiki we won't be publishing links to pre-release beta or release builds as they change frequently.
